dadguide/database_context.py

```diff
@@ -22,11 +22,14 @@
 
 
 class DbContext(object):
-    def __init__(self, database: DadguideDatabase, graph: MonsterGraph, dungeon: DungeonContext):
+    def __init__(self, database: DadguideDatabase, graph: MonsterGraph, dungeon: DungeonContext,
+                 debug_mons: Optional[List[int]] = False):
```

Comment:
shouldnt it be None if it's not debug mode? (also I wouldn't abbreviate this at all, just call it debug_monsters)